service/frontend/errors.go

```diff
@@ -53,6 +53,7 @@ var (
 	errWorkflowTypeTooLong                                = serviceerror.NewInvalidArgument("WorkflowType length exceeds limit.")
 	errWorkflowIDTooLong                                  = serviceerror.NewInvalidArgument("WorkflowId length exceeds limit.")
 	errSignalNameTooLong                                  = serviceerror.NewInvalidArgument("SignalName length exceeds limit.")
+	errSignalNameIsEmpty                                  = serviceerror.NewInvalidArgument("SignalName is empty.")
```

Comment:
The error is already define on L35. `errSignalNameNotSet`

Reply:
Thanks for pointing that out. I will make a change and update it soon.